index.js

```diff
@@ -980,6 +980,54 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] | String -> Maybe [a] | Maybe String
```

Comment:
Yup, the implication that all array functions support strings was something I was wary of; I know there was mention of it in `indexOf` PR on Ramda too.